src/test/java/com/commercetools/pingpong/controller/GameControllerTest.java

```diff
@@ -0,0 +1,26 @@
+package com.commercetools.pingpong.controller;
+
+import com.commercetools.pingpong.model.Game;
+import com.commercetools.pingpong.model.Player;
+
+import org.junit.Test;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import static org.junit.Assert.*;
+
+public class GameControllerTest {
+
+    @Test
+    public void shouldReturnBubbleMessage() {
+        Player playerTestOne = new Player("Bob");
+
+        playerTestOne.setBeginningServe();
+
+        Player playerTestTwo = new Player("Kelso");
+
+        Game matchTest = new Game(playerTestOne, playerTestTwo);
+        GameController gameControllerTest = new GameController();
+
+    }
```

Comment:
Hmm I don't see any assertions here. What are you testing here exactly?

Reply:
Nothing at the moment 😅 
But I will add an assertion after the other pull request is merged...it will test the bubbleMessage,  but i've refactored a lot of things there...

Reply:
Ah ok, which PR should be reviewed first then?

Reply:
This one: 😉 
https://github.com/TimonRey/PingPongApp/pull/20